plugins/biolatency/src/index.tsx

```diff
@@ -0,0 +1,176 @@
+import { Plugin } from '@kinvolk/headlamp-plugin/types/plugin/index.d';
+import Registry from '@kinvolk/headlamp-plugin/types/plugin/registry.d';
+import playCircleOutline from '@iconify/icons-mdi/play-circle-outline';
+import stopIcon from '@iconify/icons-mdi/stop'
+const pluginLib = window.pluginLib;
+const {Loader} = window.pluginLib.CommonComponents;
+const React = window.pluginLib.React;
+//const K8s = pluginLib.K8s.ResourceClasses;
+const { Box, IconButton, Typography, Button } = pluginLib.MuiCore;
+const { Icon } = pluginLib.Iconify;
+
+// const BIOLATENCY_CONTAINER_WIDTH = '500px';
+// const BIOLATENCY_CONTAINER_HEIGHT = '500px';
+const TRACES_RESOURCE_NAME = 'gadget.kinvolk.io';
+const IG_TRACE_NAMESPACE = 'gadget';
+const TRACE_BIOLATENCY_RESOURCE_NAME = 'biolatency';
+function prepareAPiForCustomResource() {
+    return pluginLib.apiProxy.apiFactoryWithNamespace(TRACES_RESOURCE_NAME, 'v1alpha1', 'traces')
+}
+
+function startLatencyCheck() {
+    return prepareAPiForCustomResource().patch([{"op": "add", 
+    "path": "/metadata/annotations/gadget.kinvolk.io~1operation", 
+    "value": "start"}], IG_TRACE_NAMESPACE, TRACE_BIOLATENCY_RESOURCE_NAME)
+}
```

Comment:
This can be abstracted into a class , like we're doing for `crd` and others in Headlamp.
A `patch` method can be added to the `KubeObject` class, so it can be used by other instances too.